src/Facebook/InstantArticles/Elements/Element.php

```diff
@@ -86,4 +88,30 @@ protected function emptyElement($document)
         $fragment->appendChild($document->createTextNode(''));
         return $fragment;
     }
+
+    /**
+     * Method that returns the status of ignoring empty for this Element.
+     * @since v1.1.1
+     * @return boolean true for ignore, false otherwise.
+     */
+    public function isEmptyIgnored()
+    {
+        return $this->ignore_empty;
+    }
+
+    /**
+     * Marks this Paragraph to be ignored on isValid if it is empty.
+     */
+    public function enableIgnoreEmpty()
```

Comment:
I think it is more clear to simply use `setIgnoreEmpty(true/false);`, the words "enable" and "ignore" in the same method name are confusing when reading.